packages/ui/src/components/va-input-wrapper/VaInputWrapper.vue

```diff
@@ -2,9 +2,13 @@
   <div
```

Comment:
<img width="796" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/23530004/04909926-6cbe-407f-a9ee-c4dbe5d519ab">

Keyboard focus is incorrect

Reply:
And select is opened by default for some reason.